src/scripting_api/error.js

```diff
@@ -0,0 +1,23 @@
+/* Copyright 202O Mozilla Foundation
```

Comment:
Eh, what year is that? :-)

Reply:
Calixte is trying to forget this year ;)

Reply:
ah ah... I made the typo on purpose: to have an other PR to do to fix it for the HacktoberFest ;)